tests/__init__.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+import zipfile
+import tarfile
+import os.path
+import logging
+import pytest
+from pathlib import Path
+from pprint import pformat
+from deepdiff import DeepDiff
+
+from spark_log_parser.parsing_models.application_model_v2 import SparkApplication
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
+
+# All the top-level keys that we would expect to be present in the JSON representation of a parsed SparkApplication
+PARSED_KEYS = [
+    "accumData",
+    "executors",
+    "jobData",
+    "metadata",
+    "sqlData",
+    "stageData",
+    "taskData",
+]
+
+# In parsed log files (i.e. SparkApplication.to_dict()), this is the set of top-level keys for which we would expect
+# members to not necessarily always be ordered the same way
+UNORDERED_PARSED_KEYS = [
+    "accumData",
+    "sqlData",
+    "executors"
+]
+
+APP_NAME_INCORRECT_MESSAGE = "Application name does not match expected value"
+PARSED_KEYS_MISSING_MESSAGE = "Not all keys present in parsed eventlog"
+
+
+def assert_all_files_identical(parsed_files):
```

Comment:
perhaps use "equivalent" instead of "identical" if they are different but "meaningfully the same"

Reply:
yeah, good point! originally it _was_ "identical", but that didn't remain true after a few iterations :)